kanidm_client/tests/default_entries.rs

```diff
@@ -0,0 +1,622 @@
+#![deny(warnings)]
+use std::collections::HashSet;
+
+use kanidm_client::KanidmClient;
+use kanidm_proto::v1::{Filter, Modify, ModifyList};
+
+mod common;
+use crate::common::{run_test, ADMIN_TEST_PASSWORD};
+
+static USER_READABLE_ATTRS: [&str; 9] = [
+    "name",
+    "spn",
+    "displayname",
+    "class",
+    "memberof",
+    "uuid",
+    "gidnumber",
+    "loginshell",
+    "ssh_publickey",
+];
+static SELF_WRITEABLE_ATTRS: [&str; 7] = [
+    "name",
+    "displayname",
+    "legalname",
+    "radius_secret",
+    "primary_credential",
+    "ssh_publickey",
+    "unix_password",
+];
+static DEFAULT_HP_GROUP_NAMES: [&str; 22] = [
```

Comment:
I think you don't need the size spec (22) here because you have a statically defined array it's implied?

Reply:
I think I cannot do in other way. Example [here](https://stackoverflow.com/questions/23810032/how-to-specify-const-array-in-global-scope-in-rust).

Reply:
Ohh wow, okay, ignore my comment then!